src/Pages/Withdrawals.razor

```diff
@@ -586,23 +586,15 @@
     {
         if (walletWithdrawalRequest.RequestMetadata != null)
         {
-            try
+            var requestMetadata = JsonConvert.DeserializeObject<Dictionary<string, string>>(walletWithdrawalRequest.RequestMetadata);
+            if (requestMetadata != null && requestMetadata.TryGetValue("userName", out var userName))
             {
-                var requestMetadata = JsonConvert.DeserializeObject<Dictionary<string, string>>(walletWithdrawalRequest.RequestMetadata);
-
-                if (requestMetadata.TryGetValue("userName", out var userName))
+                if (ValidationHelper.IsValidEmail(userName))
                 {
-                    if (ValidationHelper.IsValidEmail(userName))
-                    {
-                        int atIndex = userName.IndexOf('@');
-                        return atIndex >= 0 ? userName.Substring(0, atIndex) : userName;
-                    }
-                    return userName;
+                    int atIndex = userName.IndexOf('@');
+                    return atIndex >= 0 ? userName.Substring(0, atIndex) : userName;
                 }
-            }
-            catch (JsonReaderException ex)
-            {
-                Console.WriteLine("Invalid JSON: " + ex.Message);
+                return userName;
             }
         }
 
```

Comment:
I suggest improving the code to be more readable:

 return walletWithdrawalRequest.UserRequestor?.ToString() ?? string.Empty;

Reply:
Similar, defer to an issue.